src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -725,7 +747,7 @@ def custom_command(self, name, method_name, **kwargs):
                             would cause a loss of data. (bool)
             - exception_handler: Exception handler for handling non-standard exceptions (function)
             - no_wait_param: The name of a boolean parameter that will be exposed as `--no-wait` to skip long
-              running operation polling. (string)
+              running operation polling OR a callable taking one boolean argument that returns a dictionary.
```

Comment:
I'm confused by the callable thing. Where is the example of where this is used?

Reply:
Anywhere you see `no_wait_param=no_wait_params` because no_wait_params() is a method.

e.g.
https://github.com/Azure/azure-cli/pull/5584/files#diff-91f1f79fb5cebd957f4940b4a4d17635R8

https://github.com/Azure/azure-cli/pull/5584/files#diff-5241dc629edabeefd74a7984150494bbR232